src/components/media-player/ha-media-player-toggle.ts

```diff
@@ -0,0 +1,89 @@
+import { type CSSResultGroup, LitElement, css, html } from "lit";
+import { customElement, property } from "lit/decorators";
+import { mdiSpeaker } from "@mdi/js";
+
+import type { HomeAssistant } from "../../types";
+import { computeStateName } from "../../common/entity/compute_state_name";
+import { fireEvent } from "../../common/dom/fire_event";
+
+import "../ha-switch";
+import "../ha-svg-icon";
+
+@customElement("ha-media-player-toggle")
+class HaMediaPlayerToggle extends LitElement {
+  @property({ attribute: false }) public hass!: HomeAssistant;
+
+  @property({ attribute: false }) public entityId!: string;
+
+  @property({ type: Boolean }) public checked = false;
+
+  @property({ type: Boolean }) public disabled = false;
+
+  protected render() {
+    const stateObj = this.hass.states[this.entityId];
+
+    return html`<div class="list-item">
+      <ha-svg-icon .path=${mdiSpeaker}></ha-svg-icon>
+      <div class="info">
+        <div class="main-text">${computeStateName(stateObj)}</div>
+        <!-- TODO: or state if not playing -->
+        <div class="secondary-text">
+          ${stateObj.state === "playing"
+            ? stateObj.attributes.media_title
+            : this.hass.localize("ui.card.media_player.idle")}
```

Comment:
We could use the state of the player instead, to handle non-idle cases (they might be unavailable or offline?). But I can't see that the state of media players are used in such a way elsewhere.

Essentially, I'd like to copy the behavior of some "state-visualizing" components that do something like this:

```ts
this.hass.localize(`ui.card.media_player.state.${stateObj.state}`);
```